modules/c/examples/code_snippets/main.c

```diff
@@ -1078,3 +1197,392 @@ Placeholder for Date accessors.
+    // tag::replication-set-maxattemptwaittime[]
+    config.maxAttemptWaitTime = 600; //  <.>
+    // end::replication-set-maxattemptwaittime[]
+
+    // end::replication-retry-config[]
+
+    // tag::basic-authentication[]
+    // Configure Client Authentication to Basic Authentication
+    // Providing username and password credentials
+    if(docs_example_ShowBasicAuth) {
+        CBLAuthenticator* basicAuth =
+            CBLAuth_CreatePassword(FLSTR("username"),
+                                   FLSTR("passwd"));
+        config.authenticator = basicAuth; // <.>
+    }
+    // end::basic-authentication[]
+
+    // tag::session-authentication[]
+    if(docs_example_ShowSessionAuth) {
+        CBLAuthenticator* sessionAuth =
+            CBLAuth_CreateSession(FLSTR("904ac010862f37c8dd99015a33ab5a3565fd8447"),
+                                  FLSTR("optionalCookieName"));
+        config.authenticator = sessionAuth; // <.>
+    }
+
+    // end::session-authentication[]
+
+    // tag::replication-custom-header[]
+
+    // Optionally, add custom headers
+    FLMutableDict customHdrs = FLMutableDict_New();
+    FLMutableDict_SetString(customHdrs,
+                            FLSTR("customHeaderName"),
+                            FLSTR("customHeaderValue"));
+
+    config.headers = customHdrs;
+
+    // end::replication-custom-header[]
+
+    // FILTERS
+
+    // tag::replication-push-filter[]
+    // tag::replication-pull-filter[]
+    // Purpose - Illustrate use of push and-or pull filter functions
+
+    config.pushFilter = simpleReplicationFilter;
+
+    config.pullFilter = simpleReplicationFilter;
+
+    // end::replication-pull-filter[]
+    // end::replication-push-filter[]
+
+
+    //  Auto-purge over-ride
+    // tag::autopurge-override[]
+    config.disableAutoPurge = true; // <.>
+
+    // end::autopurge-override[]
+    // tag::[]
+
+    // Initialize replicator with created config
+    CBLReplicator* replicator =
+        CBLReplicator_Create(&config, &err); // <.>
+
+    // end::[]
+
+    CBLEndpoint_Free(target);
+
+    // Add optional change listener
+    CBLListenerToken* token_ReplChangeListener =
+        CBLReplicator_AddChangeListener(replicator,
+                                        simpleChangeListener,
+                                        NULL); // <.>
+
+
+}
+// END replication.html >> configure section
+
+
+
+
+// PAGE=Data Sync >> Initialize section
+// URL=https://docs.couchbase.com/couchbase-lite/current/c/replication.html#lbl-init-repl
+static CBLReplicator* docs_act_replication_Intialize(
+                        void* context,
+                        CBLReplicatorConfiguration argConfig,
+                        bool argResetRequired)
+{
+
+    CBLError err;
+
+    bool docs_example_resetRequired = argResetRequired;
+    // tag::p2p-act-rep-start-full[]
+
+    CBLReplicator* thisRepl =
+        CBLReplicator_Create(&argConfig, &err); // <.>
+
+    // end::p2p-act-rep-start-full[]
```

Comment:
Why is there a single line that does not start a replication inside of a tag called "rep-start"?

Reply:
See previous long-winded reply

Reply:
I understand how the tags work, but what I don't get is have things changed since I last worked on this?  It seems like this would create a one line snippet which tag doesn't seem to match its content.